app/components/DetailsDropdownButton.tsx

```diff
@@ -18,7 +18,10 @@ export default function DetailsDropdownButton({
   return (
     <Button type="button" {...props}>
       {children}
-      <Icon.ExpandMore role="presentation" />
+      <Icon.ExpandMore
+        role="presentation"
+        className="margin-top-neg-1 bottom-aligned"
+      />
```

Comment:
What does this change do?

Reply:
This one fixes the sizing so that the height of the button with an Icon is the same as one without